test/test-integration.js

```diff
@@ -0,0 +1,174 @@
+/* global marionette, suite, test */
+
+'use strict';
+
+var assert = require('chai').assert;
+marionette.plugin('helper', require('marionette-helper'));
+
+marionette('gaia-dialog', function() {
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        // Disable first time run UI
+        'browser.feeds.showFirstRunUI': false,
+        // Disable default browser check
+        'browser.shell.checkDefaultBrowser': false,
+        // Disable UI tutorial
+        'browser.uitour.enabled': false,
+        // Enable chrome debugging
+        'devtools.chrome.enabled': true,
+        'devtools.debugger.remote-enabled': true,
+
+        // Load integration test page on startup
+        'startup.homepage_welcome_url': __dirname + '/test-integration.html',
+
+        // Allow loading test resources oudside of test/ directory
+        // (e.g. bower-components)
+        'security.fileuri.strict_origin_policy': false,
+
+        // Enable web components
+        'dom.webcomponents.enabled': true,
+        // Enable touch events
+        'dom.w3c_touch_events.enabled': 1
+      }
+    },
+    desiredCapabilities: {
+      raisesAccessibilityExceptions: true
+    }
+  });
+
+  var dialogs = [
+    { selector: '#d1' },
+    { selector: '#d2' },
+    { selector: '#d3' },
+    { selector: '#d4' }
+  ];
+
+  function openDialog(selector) {
+    client.executeScript(function(selector) {
+      document.querySelector(selector).wrappedJSObject.open();
```

Comment:
eslint fails if i one-line it and remove {}. If {} are left in place, line's too long too..